plugins/woocommerce/changelog/update-32509_deasync_issue

```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: update
+Comment: Simply updating a patch version of the deasync package
```

Comment:
Hey @louwie17 , I actually opted _not_ to leave a changelog in the jetpack changelogger utility, but it still requests a comment. I think this is the reason for the `Comment:` part of the changelog.

That said you're right that it might be worthwhile leaving a changelog that mentions the issue, so I just did that in 3b3fb00d046ecaf2f87de8cd08dc704422ec00f5